sw/device/lib/dif/dif_usbdev.c

```diff
@@ -173,12 +173,24 @@ static bool is_valid_toggle(dif_toggle_t val) {
   return val == kDifToggleEnabled || val == kDifToggleDisabled;
 }
 
+/**
+ * Checks if the given value is a valid endpoint number.
+ */
+OT_WARN_UNUSED_RESULT
+static uint8_t endpoint_number(uint8_t endpoint) { return (endpoint & 0x7F); }
+
+/**
+ * Checks if the given value is a valid endpoint number.
+ */
```

Comment:
nit: these comments should be updated.

Reply:
I switched over to a struct with bit fields for the functions where it's relevant. Let me know if a different style is preferred.